.github/workflows/release.yml

```diff
@@ -0,0 +1,47 @@
+name: Release
+
+on:
+    workflow_dispatch:
+        branches:
+            - "main"
+    push:
+        branches:
+            - "next"
```

Comment:
We may not want to have push on branch `main` (stable) and `next` (alpha) in the same workflow.
How do we distinguish them? Like `npm i @frontify/arcade@latest` should install a stable release and the `npm i @frontify/arcade@next` should install the alpha.